### ISSUE TITLE
Add ability to navigate between build results in Trends

### ISSUE THREAD

**Issue by IgorDodukh** [fescobar/allure-docker-service#75]
Hello, 

When I have a report with results from 2 or more builds I am unable to navigate between these builds by clicking on column in Trends. For Example I want to open the previous report but it's impossible.
As I know, such feature exists in Jenkins plugin for Allure report.

Please let me know if it possible to implement this in your version of Allure report.

Thanks in advance.

**Comment by fescobar** [comment]
@IgorDodukh can you provide some screenshots to reproduce the issue, please?

**Comment by IgorDodukh** [comment]
@fescobar Here is the screenshot where I marked the column with the results I want to open.
<img width="1236" alt="Screen Shot 2020-04-23 at 2 45 38 PM" src="https://user-images.githubusercontent.com/17238214/80095920-6d505000-8571-11ea-96f5-df659194f48e.png">

**Comment by fescobar** [comment]
@IgorDodukh Yes, you are right. Give me a few days to see if it's possible to add that feature easily. Thanks

**Comment by fescobar** [comment]
@IgorDodukh  Work in Progress...

**Comment by IgorDodukh** [comment]
> @IgorDodukh Work in Progress...

Thank you. Do you have an estimate of when it can be ready?

**Comment by fescobar** [comment]
@IgorDodukh 1 week for beta, 2 weeks for release.

**Comment by fescobar** [comment]
Hi @IgorDodukh. I've released a `beta` version `frankescobar/allure-docker-service:beta`. Please if you have any other `beta` version remove your previous images and download the latest `beta`

I've implemented your feature requested. For your change, I needed to deprecate the port `4040`, just you need to use the port `5050` like this:

```
version: '3'
services:
  allure:
    image: "frankescobar/allure-docker-service:beta"
    environment:
      CHECK_RESULTS_EVERY_SECONDS: 1
      KEEP_HISTORY: "TRUE"
    ports:
      - "5050:5050"
    volumes:
      - ${PWD}/allure-results:/app/allure-results
```

Also, instead of using the url http://localhost:4040, you should use http://localhost:5050/latest-report.
https://github.com/fescobar/allure-docker-service/tree/beta#port-4040-deprecated

The documentation is up to date in the README.md from `beta` branch
https://github.com/fescobar/allure-docker-service/tree/beta

Here is how it works the navigability between build results in trends
https://github.com/fescobar/allure-docker-service/tree/beta#keep-history-and-trends
Before releasing this, I will add a way to keep just a specific quantity of executions to avoid overload the storage in the machine.

Also if you want to check how it works the new feature `Multiple Projects` you can follow this section:
https://github.com/fescobar/allure-docker-service/tree/beta#MULTIPLE-PROJECTS---REMOTE-REPORTS

I appreciate your suggestions and please let me know if everything is working well or not.

**Comment by IgorDodukh** [comment]
@fescobar Thank you for your work. I will test new build  and provide you my feedback in a few days.

**Comment by IgorDodukh** [comment]
Hi @fescobar. 

I've tested new `beta` version of `allure-docker-service` and it looks great for me. 
I tried to work with all new features and everything works as expected. No issues were noticed.

The only improvement I'd like to request:
- GitLab support in `execution_type` (GitLab logo)

I appreciate your efforts. Thank you!

**Comment by fescobar** [comment]
Thanks @IgorDodukh, just you can use the icons provided for the native allure framework:

https://github.com/allure-framework/allure2/tree/master/allure-generator/src/main/javascript/blocks/executor-icon

**Comment by fescobar** [comment]
@IgorDodukh  The tests were successful!!. I've re-released version `2.13.3`, please remove your local images if you are using the same version to pull the latest `2.13.3`.
Check the documentation about the new changes here:
https://github.com/fescobar/allure-docker-service#allure-docker-service-1
https://github.com/fescobar/allure-docker-service/releases/tag/v2.13.3
